.eslintrc.json

```diff
@@ -12,25 +12,38 @@
     "globalThis": false
   },
   "rules": {
-    "complexity": ["error", 20],
     "implicit-arrow-linebreak": "off",
     "function-paren-newline": "off",
     "arrow-parens": "off",
     "strict": "off",
-    "max-depth": ["error", { "max": 5 }],
-    "max-lines": ["error", 400],
-    "max-nested-callbacks": ["error", 4],
-    "max-statements": ["error", 40],
-    "max-statements-per-line": ["error", { "max": 1 }],    "no-console": "off",
-    "no-unused-vars": ["error", { "argsIgnorePattern": "^_" }],
+    "prefer-destructuring": "off",
+    "no-else-return": "off",
+    "no-console": "off",
+    "no-unused-vars": [
+      "error",
+      {
+        "argsIgnorePattern": "^_",
+        "varsIgnorePattern": "^_"
+      }
+    ],
     "no-return-assign": "off",
     "no-param-reassign": "off",
-    "no-restricted-syntax": ["off", "ForOfStatement"],
+    "no-restricted-syntax": [
+      "off",
+      "ForOfStatement"
+    ],
     "no-unused-expressions": "off",
     "no-loop-func": "off",
+    "no-inner-declarations": "off",
+    "import/prefer-default-export": "off",
+    "complexity": ["error", 20],
```

Comment:
So long as we are disturbing the order, perhaps we should sort these keys.

This command, using `jq`, will do this automatically.

```sh
jq '. + {rules: (.rules | to_entries | sort_by(.key) | from_entries)}'
```

Provided `.eslintrc.json` for input and output. I don’t know your preferred idiom for transforming a file in place, but please share. Otherwise, on a Mac you can do this and avoid a temp file:

```sh
$CMD < .eslintrc.json | pbcopy; pbpaste > .eslintrc.json
```

Or, in automation I prefer to take advantage of the content address store we all have handy:

```sh
HASH=$($CMD < $FILE | git hash-object -w --stdin)
git cat-file blob "$HASH" > $FILE
```

Reply:
Actually, at this time, my intention on disturbing the order is that the first N look the same as the eslint settings in agoric-sdk; and that all the ones not in agoric-sdk come after the ones in agoric-sdk. Eventually, we should do one more sensible thing identically on both sides. But this PR is only incremental improvement towards that.